understanding/22/dragging-movements.html

```diff
@@ -48,8 +48,10 @@ <h2>Intent of Dragging Movements</h2>
       <p>Some tasks may typically involve dragging, such as sorting items in a list, but as there are other ways to achieve that task they are not considered essential.</p>
       
       <h3>Distinguishing dragging movements from path-based pointer gestures</h3>
-      <p>A dragging movement is a pointer interaction where only the endpoints matter. Once the pointer engages with a target, the target (or a position mark related to the target) will follow the pointer regardless of the direction of the pointer movement. The target's movement may be constrained to one axis and often, to a particular range on that axis, but within those constraints, the x or y position of the target will mirror the corresponding x or y position of the pointer until the pointer disengages the target.</p> 
-      <p>In contrast to dragging movements, path-based gestures involve at least an initial directionality of the pointer movement. This is defined by stating that the path has to go through at least one intermediate point to qualify as pointer gesture. The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a>.</p>
+      <p>A dragging movement is a pointer interaction where only the endpoints matter. Once the pointer engages with a target, the target (or a position mark related to the target) will follow the pointer regardless of the direction of the pointer movement. The target's movement may be constrained to one axis and often, to a particular range on that axis, but within those constraints, the x or y position of the target will mirror the corresponding x or y position of the pointer until the pointer disengages the target. So dragging is a motion between the start and end points without an intermediate point which can go in any direction and is entirely non-prescriptive. </p> 
+      <p>In contrast to dragging movements, path-based gestures involve at least an initial directionality of the pointer movement. This is defined by stating that the path has to go through at least one intermediate point to qualify as pointer gesture. The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. So, although a user could drag in the same way as a gesture, the difference is that a gesture requires an intermediate point, dragging does not. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a>.</p>
```

Comment:
In my view, the sentences added: "The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. So, although a user could drag in the same way as a gesture, the difference is that a gesture requires an intermediate point, dragging does not" make the text more verbose without providing more clarity. I would leave the text as it is.